src/lib/holocene/table/table-header-row.svelte

```diff
@@ -1,5 +1,5 @@
 <tr class={$$props.class}>
-  <th />
+  <td />
   <slot />
-  <th />
+  <td />
```

Comment:
I'm going to try to play with getting rid of these entirely 😈

Reply:
💯 % behind that endeavor